### PR TITLE
batch handle tinc of perfcounters

### PR DESCRIPTION
**Pull request by majianpeng**
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/msg/async/ProtocolV1.cc [hunk 1]
```diff
@@ -365,11 +363,12 @@ void ProtocolV1::write_event() {
         ack_left -= left;
         left = ack_left;
         r = connection->_try_send(left);
-      } else if (is_queued()) {
+      } else if (connection->is_queued()) { // data lefted in last write_event
         r = connection->_try_send();
       }
     }
 
+    connection->logger->tinc(l_msgr_send_messages_queue_lat, queue_lat);
```
Comment (rjfd) [inline src/msg/async/ProtocolV1.cc]: I don't think doing a single `tinc` call with the sum of the latencies is equivalent to make several `tinc` calls with each individual latency value. See perf_counter implementation here: https://github.com/ceph/ceph/blob/master/src/common/perf_counters.cc#L243

**Comment by majianpeng** [comment]
@rjfd, update, please review. Thanks very much.  @tchaikov , please review.

**Comment by rjfd** [comment]
jenkins test this please

**Comment by majianpeng** [comment]
jenkins test this please

**Comment by majianpeng** [comment]
@rjfd  ping.

**Comment by majianpeng** [comment]
@rjfd  update by your suggestion. pls review. Thanks

**Comment by majianpeng** [comment]
@rjfd. ping!

**Comment by majianpeng** [comment]
retest this please

**Pull request closed without merging**